### PR TITLE
added cards for TTbarDMJets sample for 14TeV HL-LHC

### PR DESCRIPTION
**Pull request by ksbeerna**

### CODE REVIEW

**Comment by efeyazgan** [comment]
please remove the commented lines

File: bin/MadGraph5_aMCatNLO/cards/production/2017/14TeV/TTbarDMJets/TTbarDMJets_DiLept_NLO_scalar_Mchi_1_Mphi_100/ttbarDM__DMsimp_s_spin0__mchi_1_mphi_100_gSM_1_gDM_1_7000GeV_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+#***********************************************************************
+ 1   = lpp1    ! beam 1 type (0 = no PDF)
+ 1   = lpp2    ! beam 2 type (0 = no PDF)
+ 7000  = ebeam1  ! beam 1 energy in GeV
+ 7000  = ebeam2  ! beam 2 energy in GeV
+#***********************************************************************
+# PDF choice: this automatically fixes also alpha_s(MZ) and its evol.  *
+#***********************************************************************
+ lhapdf = pdlabel ! PDF set
+ # NNPDF30_nlo_as_0118_hessian
```
Comment (efeyazgan) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/14TeV/TTbarDMJets/TTbarDMJets_DiLept_NLO_scalar_Mchi_1_Mphi_100/ttbarDM__DMsimp_s_spin0__mchi_1_mphi_100_gSM_1_gDM_1_7000GeV_run_card.dat]: please remove this comment

Comment (ksbeerna) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/14TeV/TTbarDMJets/TTbarDMJets_DiLept_NLO_scalar_Mchi_1_Mphi_100/ttbarDM__DMsimp_s_spin0__mchi_1_mphi_100_gSM_1_gDM_1_7000GeV_run_card.dat]: Sorry, overlooked that. Thank you for checking!

File: bin/MadGraph5_aMCatNLO/cards/production/2017/14TeV/TTbarDMJets/TTbarDMJets_DiLept_NLO_pseudoscalar_Mchi_1_Mphi_100/ttbarDM__DMsimp_ps_spin0__mchi_1_mphi_100_gSM_1_gDM_1_7000GeV_proc_card.dat [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import model DMsimp_ps_spin0
+
+define p = g u c d s b u~ c~ d~ s~ b~
+define j = p
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/14TeV/TTbarDMJets/TTbarDMJets_DiLept_NLO_pseudoscalar_Mchi_1_Mphi_100/ttbarDM__DMsimp_ps_spin0__mchi_1_mphi_100_gSM_1_gDM_1_7000GeV_proc_card.dat]: we suggest to remove proton and jet definition for 2017 datacards

**Pull request merged**